cosmo_tester/test_suites/cluster/conftest.py

```diff
@@ -489,36 +491,11 @@ def _bootstrap_lb_node(node, managers, tempdir, logger):
     node.run_command('sudo /tmp/haproxy_install.sh')
 
     # configure haproxy
-    haproxy_config = """global
-    maxconn 100
-    tune.ssl.default-dh-param 2048
-defaults
-    log global
-    retries 2
-    timeout client 30m
-    timeout connect 4s
-    timeout server 30m
-    timeout check 5s
-listen manager
-    bind *:80
-    bind *:443 ssl crt /etc/haproxy/cert.pem
-    redirect scheme https if !{ ssl_fc }
-    mode http
-    option forwardfor
-    stick-table type ip size 1m expire 1h
-    stick on src
-    option httpchk GET /api/v3.1/status
-    http-check expect status 401
-    default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
-    """
-
-    for manager in managers:
-        haproxy_config += '    server manager_{private} {public} maxconn ' \
-                          '100 ssl check-ssl port 443 ca-file /etc/haproxy/' \
-                          'ca.crt\n'.format(private=manager.private_ip_address,
-                                            public=manager.ip_address)
-    node.run_command('echo "{config}" | sudo tee '
-                     '/etc/haproxy/haproxy.cfg'.format(config=haproxy_config))
+    template = Environment(
+        loader=FileSystemLoader(CONFIG_DIR)).get_template('haproxy.cfg')
+    config = template.render(managers=managers)
+    node.run_command('echo \'{config}\' | sudo tee '
```

Comment:
I think we have a method for uploading files somewhere, which would be better to use, in case we put quotes in the config.